pontoon/base/models.py

```diff
@@ -3430,16 +3430,19 @@ def calculate_stats(self, save=True):
 class Comment(models.Model):
     author = models.ForeignKey(User)
     timestamp = models.DateTimeField(default=timezone.now)
-    translation = models.ForeignKey(Translation)
+    translation = models.ForeignKey(Translation, related_name='comments')
     content = models.TextField()
 
     def __str__(self):
         return self.content
 
     def serialize(self):
         return {
-            'author': self.author_id,
-            'timestamp': self.timestamp,
-            'content': self.content,
-            'translation': self.translation_id
+            "author": self.author.name_or_email,
+            "username": self.author.username,
+            "userGravatarUrlSmall": self.author.gravatar_url(88),
```

Comment:
Ah, here's the trick: Python variables should use `underscore_case`. :) This is confusing, sorry. We effectively have two different languages with different naming conventions. We want to strictly enforce those naming conventions. The only time when they conflict is through the API. And that's where we need some specific code to manually convert those names.

So, here you should revert this change. But keep your type definition in JS. Then on the API function, in front-end, where you get the data, you should have some logic to transform the data, mapping one convention (underscore_case from Python) to the other (camelCase for JS).

Here's where the data gets fetched: https://github.com/mozilla/pontoon/blob/6cc116bfff32d1c9ac7133535d4cb8055074b5a2/frontend/src/core/api/entity.js#L115

_Side note: if this data was exposed with GraphQL, we wouldn't have that problem, as it handles that conversion automatically for us. Sadly, this is not the case yet, but doing this conversion now means it will be easier later on to switch to fetching that data through GraphQL._